functions.opengraph.php

```diff
@@ -100,10 +104,12 @@ function jetpack_og_tags() {
 
 		$tags['og:url']         = get_permalink( $data->ID );
 		if ( ! post_password_required() ) {
-			if ( ! empty( $data->post_excerpt ) ) {
-				$tags['og:description'] = preg_replace( '@https?://[\S]+@', '', strip_shortcodes( wp_kses( $data->post_excerpt, array() ) ) );
+			$excerpt = get_the_excerpt( $data );
```

Comment:
I like `get_the_excerpt` better than what I removed in #2899 because, from what I've seen, the problematic stuff is with `the_excerpt` filter. Since this avoids that, it would avoid specifically the likes and sharedaddy problems with the_excerpt.